trapper/metrics/input_handlers/token_classification_input_handler.py

```diff
@@ -48,6 +48,7 @@ def preprocess(self, eval_pred: EvalPrediction) -> EvalPrediction:
             actual_labels.append(actual_label)
 
         processed_eval_pred = EvalPrediction(
-            predictions=predictions, label_ids=references
+            predictions=np.array(actual_predictions),
+            label_ids=np.array(actual_labels),
```

Comment:
Is this a bug fix?

Reply:
Yes, the former version was just ignoring the preprocess step.